leobot_description/robot/robot.urdf.xacro

```diff
@@ -14,6 +14,60 @@
     <xacro:cylinder_collision radius="0.25" height="1.0"/>
     <xacro:cylinder_inertia mass="10" radius="0.25" height="1.0"/>
   </link>
+       <!-- Leo_left_wheel -->
+    <link name="left_wheel_link">
+    <visual>
+      <xacro:cylinder_geometry radius="0.1" height="0.05"/>
+      <xacro:red_material/>
+    </visual>
+    <xacro:cylinder_collision radius="0.1" height="0.05"/>
+    <xacro:cylinder_inertia mass="10" radius="0.1" height="0.05"/>
+  </link>
+
+    <!-- left_wheel-body -->
+  <joint name="left_wheel_body_joint" type="continuous">
+    <origin rpy="1.570795 0 0" xyz="0.0 0.275 -0.55"/>
```

Comment:
Pi is predefined constant. Please use XACRO expressions to calculate 90 degrees as here http://wiki.ros.org/xacro#Math_expressions